_includes/script.html

```diff
@@ -1,2 +1,3 @@
 <!-- Scripts -->
+<script src="https://unpkg.com/simple-jekyll-search/dest/simple-jekyll-search.min.js"></script>
```

Comment:
Since we’re making an effort to use a privacy conscious and « sovereign » way of implementing the search, maybe go all the way and not use an external CDN?

Reply:
Changed. I copied the script into assets to avoid importing the whole npm/node stack